src/MapViewController.swift

```diff
@@ -128,6 +141,49 @@ public class MapViewController: TGMapViewController, LocationManagerDelegate, TG
     }
   }
 
+  public func display(route: OTRRoutingResult) throws {
+    //TODO: We eventually should support N number of routes.
+    if let routeMarker = currentRouteMarker {
+      //We don't throw if the remove fails here because we want to silently replace the route
+      currentRouteMarker = nil
+      markerRemove(routeMarker)
+    }
+    let routeLeg = route.legs[0]
```

Comment:
My thought is we could support it rather easily with a single poly line (just adding up the sizes of the individual legs) and then the implementing dev could just drop extra markers where they know there are stops involved, rather than having multiple polylines running around to keep track of.